src/include/splash/SerialDataCollector.hpp

```diff
@@ -61,10 +61,11 @@ namespace splash
         /**
          * Constructs a filename from a base filename and the process' mpi position
          * such as baseFilename+mpiPos+.h5
+         * Does nothing if the filename already contains a ".h5"
```

Comment:
It does nothing, if and only if the file contains ".h5". This is correct. See https://github.com/ComputationalRadiationPhysics/libSplash/pull/234/files#diff-d11d9b83ed9825564518a8526501da72L73